tests/unittests/test_subp.py

```diff
@@ -232,13 +232,17 @@ def test_c_lang_can_take_utf8_args(self):
         the default encoding will be set to ascii.  In such an environment
         Popen(['command', 'non-ascii-arg']) would cause a UnicodeDecodeError.
         """
-        python_prog = '\n'.join([
-            'import json, sys',
-            'from cloudinit.subp import subp',
-            'data = sys.stdin.read()',
-            'cmd = json.loads(data)',
-            'subp(cmd, capture=False)',
-            ''])
+        python_prog = '\n'.join(
+            [
+                'import json, sys',
+                'sys.path.append("{}")'.format(get_top_level_dir()),
```

Comment:
Setting the pythonpath environment variable in subp didn't seem to work. That would have been cleaner, but this works too.

Reply:
you probably want to prepend, not append.  The right 'cloudinit.subp' to import is the one in the top level dir, if there were others in the path for some reason then you'd get the wrong one.

that said, '' is (I think) usually the first token of sys.path, which is why this worked before.  So I don't really think you have to change anything unless the intent is to execute these tests with some other working directory.